### ISSUE TITLE
lib pruning

### ISSUE THREAD

**Issue by hymn4u** [google/filament#7594]
Hi, this repo is great firstly. And i got confused when i use filament as library, for example [v1.50.3] version, the libs memory: ios [filament-v1.50.3-ios.tgz] 59.6M; android [filamat-v1.50.3-android.aar] 9.31M; windows [filament-v1.50.3-windows.tgz] 500+M

i'm pretty fresh here, and i wanna know is that possible to pruning the ios version memory to as big as android 9M. And how can i do that？

Any help is appreciate，thanks

**Comment by romainguy** [comment]
The iOS libraries are static universal libraries, and have therefore not received the level of optimizations you get from Android's AAR files (which contain shared libraries in which the static libraries have been linked with the JNI code). When building an iOS app you should see the size go down dramatically after linking into the executable.

In addition, the iOS archive contains *all* the Filament libraries, whereas Android libraries are split across multiple AARs and do not offer access to everything (only what's exposed via Java).

tl;dr Just link against what you need on iOS, and you'll be fine.
